doc/man-template.roff

```diff
@@ -81,11 +81,15 @@ environment variable is to rename the binary to either http or https.
 .BR REQUESTS_CA_BUNDLE ", " CURL_CA_BUNDLE
 Sets a custom CA bundle path.
 .TP
+.B http_proxy
+Sets the proxy server to use for HTTP.
+.TP
 .B HTTPS_PROXY
 Sets the proxy server to use for HTTPS.
 .TP
-.B http_proxy
-Sets the proxy server to use for HTTP.
+.B NO_PROXY
+List of comma seperated host names where proxy server is ignored. Use "*"
+to match all host names.
```

Comment:
```suggestion
List of comma-separated hosts for which to ignore the other proxy environment
variables. "*" matches all host names.
```
It doesn't seem to affect the `--proxy` option.

The full rules are [annoyingly complicated](https://github.com/seanmonstar/reqwest/blob/5656fb861635a05fcd0ea32ab5251d954414a39f/src/proxy.rs#L384) and probably not worth documenting in our man page.